src/items.rs

```diff
@@ -264,6 +264,20 @@ pub fn set_priority(config: Config, item: items::Item) {
     }
 }
 
+pub fn edit_item(config: Config, item: items::Item) {
+    println!("{}", item.fmt(&config));
+
+    let new_name =
+        config::get_input("Enter a new name for this item").expect("could not get new name");
```

Comment:
Instead of expect you could use `?` as `get_input` returns `Result<String, String>`

Reply:
For this I kept the same pattern as in the [`set_priority`](https://github.com/alanvardy/tod/blob/38a165291c8a00343eb47c4beae24d510d2a6c16/src/items.rs#L244) function, to keep consistency. Should I still change it?

Reply:
Hmmm.. no. I think you can leave it as is and I'll take another look later if I should refactor both of them.

Consistency is great :+1: